subsys/bluetooth/mesh/net.c

```diff
@@ -674,8 +682,8 @@ static bool relay_to_adv(enum bt_mesh_net_if net_if)
 	}
 }
 
-static void bt_mesh_net_relay(struct net_buf_simple *sbuf,
-			      struct bt_mesh_net_rx *rx)
+static void bt_mesh_net_forward(struct net_buf_simple *sbuf,
```

Comment:
I'm not bug fun of renaming the existing functionality just because of. There are a lot of people working on it and some renaming makes it difficult to maintain. 
Also, this is the real relaying not forwarding since the direct forwarding is separate functionality, this renaming will create confusing.

Reply:
It makes perfect sense to me. But I'm fine retaining older name.

Reply:
If each maintainer start renaming local functions and variables according to some feeling, it will be maintenance hell. Let's refuse it until there is real reason for it, like functionality refactoring and etc.